content/en/user-guide/aws/acm/index.md

```diff
@@ -41,25 +36,142 @@ This command will return the Amazon Resource Name (ARN) of the new certificate,
+
+{{< command >}}
+$ awslocal acm-pca create-certificate-authority \
+     --certificate-authority-configuration '{
+        "KeyAlgorithm":"RSA_2048",
+        "SigningAlgorithm":"SHA256WITHRSA",
+        "Subject":{
+            "Country":"US",
+            "Organization":"Example Corp",
+            "OrganizationalUnit":"Sales",
+            "State":"WA",
+            "Locality":"Seattle",
+            "CommonName":"www.example.com"
+        }
+     }' \
+     --revocation-configuration '{
+        "OcspConfiguration":{
+            "Enabled":true
+        }
+     }' \
+     --certificate-authority-type "ROOT" \
+     --tags  Key=Name,Value=MyPCA
+<disable-copy>
+{
+    "CertificateAuthorityArn": "arn:aws:acm-pca:us-east-1:000000000000:certificate-authority/f38ee966-bc23-40f8-8143-e981aee73600"
+}
+</disable-copy>
+{{< /command >}}
+
+Note the `CertificateAuthorityArn` from the output as it will be needed for subsequent commands.
+
+### Describe the Certificate Authority
+
+To retrieve the detailed information about the created Certificate Authority, use the [`DescribeCertificateAuthority`](https://docs.aws.amazon.com/acm-pca/latest/APIReference/API_DescribeCertificateAuthority.html) API. This command returns the detailed information about the CA, including the CA's ARN, status, and configuration.
+
+{{< command >}}
+$ awslocal acm-pca describe-certificate-authority \
+    --certificate-authority-arn arn:aws:acm-pca:us-east-1:000000000000:certificate-authority/f38ee966-bc23-40f8-8143-e981aee73600
+<disable-copy>
+{
+    "CertificateAuthority": {
+        "Arn": "arn:aws:acm-pca:us-east-1:000000000000:certificate-authority/f38ee966-bc23-40f8-8143-e981aee73600",
+        "OwnerAccount": "000000000000",
+        "CreatedAt": "2023-12-23T20:52:20.917796+05:30",
+        "Type": "ROOT",
+        "Status": "PENDING_CERTIFICATE",
+        "CertificateAuthorityConfiguration": {
+            "KeyAlgorithm": "RSA_2048",
+            "SigningAlgorithm": "SHA256WITHRSA",
+            "Subject": {
+                "Country": "US",
+                "Organization": "Example Corp",
+                "OrganizationalUnit": "Sales",
+                "State": "WA",
+                "CommonName": "www.example.com",
+                "Locality": "Seattle"
+            }
+        },
+        "RevocationConfiguration": {
+            "OcspConfiguration": {
+                "Enabled": true
+            }
+        },
+        "KeyStorageSecurityStandard": "FIPS_140_2_LEVEL_3_OR_HIGHER",
+        "UsageMode": "SHORT_LIVED_CERTIFICATE"
+    }
+}
+</disable-copy>
+{{< /command >}}
+
+This command provides detailed information about the Certificate Authority, including its type, status, and configuration details.
```

Comment:
This sentence seems a bit redundant (considering the sentence at the top of the section).